src/main/kotlin/org/rust/ide/formatter/RustBlock.kt

```diff
@@ -58,33 +56,31 @@ class RustBlock(
             else                -> Indent.getNoneIndent()
         }
     }
+}
 
-    companion object {
-        private val BLOCK_START_TOKEN_SET = TokenSet.create(
-            PUB,
-            MOD,
-            STRUCT,
-            ENUM,
-            IMPL,
-            TRAIT,
-            MATCH
-        )
+private val BLOCK_START_TOKEN_SET = TokenSet.create(
```

Comment:
I'm not sure, but it's possible that I will delete these guys when I will be working on indentation anyway.